src/index.ts

```diff
@@ -103,10 +98,28 @@ export function getFileName(inPath: string) {
 }
 
 async function main() {
-    const renderer = new ImageRenderer(args.width, args.height, args.format, args.plddt, new FocusFirstResidue());
+    // Support multiple, comma-separated files
+    var files:string[] = args.in.split(',');
+
+    // Confirm all files exist
+    files.forEach(file => {
+        if (!fs.existsSync(file)) {
+            console.error(`Input path "${file}" does not exist`);
+            process.exit(1);
+        }
+    });
 
-    const fileName = getFileName(args.in);
-    const cif = await readCifFile(args.in);
+    // Run the rendering process
+    files.forEach(async file => {
+        await renderFile(file);
+    });
```

Comment:
```suggestion
    for (const file in files) {
        await renderFile(file);
    });
```

Need a normal loop here so the `await` does what it's supposed to. What you did would execute all the renders in parallel which you don't really want as this is a single threaded environment and you would also likely run out of memory if the input list was really long.